test-network-function/generic/suite.go

```diff
@@ -122,6 +128,87 @@ func createPartnerContainers(config *tnfConfig.TestConfiguration) map[tnfConfig.
 	return createContainers(config.PartnerContainers)
 }
 
+func pipeCmdToStd(cmd *exec.Cmd) {
+	cmd.Stdout = os.Stdout
+	cmd.Stderr = os.Stderr
+}
+
+func waitForPod() error {
+	for i := 0; i < 10; i++ {
+		checkPodStatusCmd := exec.Command("oc", "get", "pod", "node-master", "-o", "jsonpath='{.status.containerStatuses[0].ready}'")
```

Comment:
We have taken the approach of utilizing expect instead of `exec.Command`.  Please see:
https://github.com/test-network-function/test-network-function/blob/main/CONTRIBUTING.md#test-implementation-guidelines

Please Spawn a Shell similar to https://github.com/test-network-function/test-network-function/blob/main/test-network-function/container/suite.go#L60#L67 then implement the `tnf.Tester` and `reel.Handler` FSM interfaces similar to https://github.com/test-network-function/test-network-function/blob/main/pkg/tnf/handlers/ping/ping.go.